src/test/scala/nl.knaw.dans.easy.deposit/servlets/IntegrationSpec.scala

```diff
@@ -354,8 +354,12 @@ class IntegrationSpec extends TestSupportFixture with ServletFixture with Scalat
       body shouldBe ""
       status shouldBe NO_CONTENT_204
 
+      val bagStoreBagId = new PropertiesConfiguration((depositDir / "deposit.properties").toJava)
+        .getString("bag-store.bag-id")
+      bagStoreBagId shouldNot be(null)
```

Comment:
if you test this property, it probably means that you expect `bagStoreBagId` not to be present earlier. Could you add that test as well, please?